bots/image-create

```diff
@@ -184,10 +188,13 @@ class MachineBuilder:
         data_file = os.path.join(data_dir, name)
         shutil.move(partial, data_file)
 
+        # Remove temp image file
+        os.unlink(self.machine.image_file)
```

Comment:
We don't hold the file open so I don't think we can use one of the self cleaning options. We still need to remove it ourselves.